src/io.jl

```diff
@@ -167,26 +168,26 @@ function savetomographyobserver(observer::Observer, output_path::String; model =
       write(fout, "model", "nothing")
     end
     
-    params = results(observer, "parameters")
-    g1 = create_group(fout, "parameters")
-    g1["batchsize"] = params["batchsize"] 
-    g1["nshots"] = params["dataset_size"] 
-    g1["measurement_frequency"] = params["measurement_frequency"]
-    g1["optimizer"] = params["optimizer"][:name]
-    g1["learning_rate"] = params["optimizer"][:η]
-    attributes(g1)["Description"] = "This group contains the training parameters."
-    
     g2 = create_group(fout, "measurements")
-    for (measurement, value) in observer.measurements
-      if measurement != "parameters"
-        g2[measurement] = real.(last(value))
-      end
+    for (measurement, value) in observer.data 
+      # TODO HDF% cannot export complex numbers (?)
```

Comment:
So what does this do with complex numbers right now?

Reply:
Also, I guess this assumes that all measurements values are vectors of numbers, correct? Is this a safe assumption?

Reply:
I remember throwing an error for complex numbers. I can reproduce that.

This is how we define them in the observer, as vectors. Overall I'm not very satisfied with standalone HDF5 and would prefer JLD. We can think about that.

Reply:
Yeah, JLD has it's own downsides, like if you change the definition of a type after you save it I remember it was kind of complicated figuring out how to load it properly.

Reply:
Do you mean HDF5 in your comment here?